### PR TITLE
Revert "7 built a branch for the first draft of the game"

### PR DESCRIPTION
**Pull request by subtostrikeman**

### CODE REVIEW

**Review by OleMortensen8 — CHANGES_REQUESTED** [review]
this plugin should not be added into main

File: project.godot
Comment (OleMortensen8) [inline project.godot:1]: you removed the execution parameters?

File: gun.tscn
Comment (OleMortensen8) [inline gun.tscn:1]: gun.tscn is what I'll review the other stuff should be delete

File: player.tscn
Comment (OleMortensen8) [inline player.tscn:1]: player.tscn is what I'll review the other stuff should be delete

**Comment by subtostrikeman** [comment]
how do i delete all that you want me to

File: project.godot
Comment (OleMortensen8) [reply project.godot:1]: project.godot Is critical for compilation.

**Review by OleMortensen8 — CHANGES_REQUESTED** [review]
a few changes is required

**Comment by subtostrikeman** [comment]
its done

**Comment by OleMortensen8** [comment]
> how do i delete all that you want me to

you delete the plugin git-godot then i'll review main with this pull request to either approve or create further disscussion about the changes.

File: project.godot
Comment (subtostrikeman) [reply project.godot:1]: how to do that

**Comment by OleMortensen8** [comment]
@subtostrikeman  I'll look it over.

**Comment by subtostrikeman** [comment]
its done

**Review by OleMortensen8 — CHANGES_REQUESTED** [review]
Vi Skal have et møde om disse ændringer, De er en forværring af main. meeting time https://meet.jit.si/WrittenTaxisResolveAlike

**Comment by OleMortensen8** [comment]
This Is a garbage request I'll close it.

**Pull request closed without merging**